terraform/aws.tf

```diff
@@ -0,0 +1,5 @@
+provider "aws" {
+  access_key = "${var.access_key}"
```

Comment:
This can be simplified to the snippet below if you set `AWS_PROFILE` when invoking `terraform`:

```hcl
provider "aws" {
  region = "${var.region}"
}
```